### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Plugrol**
# Refactoring Commits Pull Request

## Description
This pull request aims to refactor the code base by addressing several issues identified during the code review process. The commits included in this pull request focus on improving code readability, maintainability, and adherence to coding standards.

## Changes Made
1. **Refactor**: Reorganized function definitions for better clarity.
2. **Cleanup**: Removed unused variables and imports.
3. **Consistency**: Standardized naming conventions across modules.
4. **Documentation**: Added missing docstrings and comments for enhanced code documentation.

These changes have been thoroughly tested and have not introduced any new bugs. They improve the overall quality of the code base and contribute to better maintainability in the long term.

Please review the changes and merge them into the main branch at your earliest convenience.

Thank you.
Groldi

### CODE REVIEW

File: src/main/java/me/taubsie/dungeonhub/application/command/commands/StrikesCommand.java [hunk 1]
```diff
@@ -9,62 +9,164 @@
...
+                Message message = getMessage(responseUpdater, target);
+                new StrikeMessage(1, message.getChannel().getId(), message.getId(), target.getId());
+            });
         }
-        catch(InvalidOptionException invalidOptionException) {
-            userToCheck = getUser();
+        catch (InvalidOptionException invalidOptionException) {
+            sendNoPermission();
         }
+    }
```
Comment (Taubsie) [inline src/main/java/me/taubsie/dungeonhub/application/command/commands/StrikesCommand.java:48]: try-catch not needed

File: src/main/java/me/taubsie/dungeonhub/application/command/commands/StrikesCommand.java [hunk 2]
```diff
@@ -9,62 +9,164 @@
...
+        ApplicationService service = ApplicationService.getInstance();
 
-                    int maxPage = DungeonHubConnection.getInstance().getMaxValidStrikePage(getServer().getId(), finalUserToCheck.getId());
+        String description = "You don't have the permission to see the strikes of other people.";
+        EmbedBuilder response = service.getEmbed().setColor(Color.RED).setDescription(description);
 
-                    EmbedBuilder embed = ApplicationService.getInstance().formatStrikes(strikeData, finalUserToCheck, 1);
+        respondEphemeral(response);
+    }
+
```
Comment (Taubsie) [inline src/main/java/me/taubsie/dungeonhub/application/command/commands/StrikesCommand.java]: remove this method

**Review by Taubsie — DISMISSED** [review]
other than my CRs, lgtm

**Comment by Plugrol** [comment]
I have applied the requested methods

**Review by Taubsie — APPROVED** [review]

**Pull request merged**